website/content/en/docs/concepts/disruption.md

```diff
@@ -210,7 +210,7 @@ Voluntary node removal does not include [Interruption]({{<ref "#interruption" >}
 
 ### Node-Level Controls
 
-Nodes can be opted out of consolidation disruption by setting the annotation `karpenter.sh/do-not-disrupt: "true"` on the node.
+You can block Karpenter from voluntarily choosing to disrupt certain nodes by setting the `karpenter.sh/do-not-disrupt: "true"` annotation on the node. This will prevent consolidation and expiration actions on the node.
```

Comment:
Nice catch and update! One ask: Can you update these changes across the v0.32 and v0.33 doc versions as well?

Reply:
Added the changes to those, as well as v0.30 and v0.31.

Reply:
I just called it out, but I don't think that we want this change in v0.30 or v0.31 since that annotation wasn't available there yet